pages/blog.html

```diff
@@ -0,0 +1,123 @@
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+   </head>
+   <body class="blogmain">
+      <div class = "container">
+         <!-- Navbar and Page Menu -->
+         <div class="header box">
+            <div class="logo">
+               <a class="logo" href="bloghome.html">
+               <img src="images/bloglogo.png.png" alt="Homepage">
+               </a>
+               <h1>
+                  A year of Code Blog
+               </h1>
+               <ul class="listnav">
+                  <li><a href="#">About</a></li>
+               </ul>
+               <ul class="listnav">
+                  <li><a href="blog.html"> Most Current Blog Post</a></li>
+               </ul>
+            </div>
+         </div>
+      </div>
+      <div class="blogcontent">
+         <h2> BLOG TITLE Heading </h2>
+         <h5> Title description , DATE </h5>
+         <h5> Author -- Bukunmi G </h5>
+         <img  class = "blogpostfoto" src= "images/blog2.jpg" alt="blog picture">
+         <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+            Architecto, ipsam excepturi necessitatibus similique dolores non consequuntur eos sequi, perferendis mollitia adipisci eveniet cum, 
+            impedit minima alias provident cumque ipsum quos?.Lorem 
+         </p>
+         <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus voluptatem placeat blanditiis
+            at et aliquid architecto adipisci ipsa 
+            maxime nesciunt perspiciatis quaerat labore, quidem velit vel fugiat 
+            esse id? Veniam.
+         </p>
+         <figure>
+            <img src="images/blog3.jpg" alt="coding">
+            <figcaption><b> Fig.1 - Me Procastinating before I start coding away.</b></figcaption>
+         </figure>
+         <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus voluptatem placeat blanditiis
+            at et aliquid architecto adipisci ipsa 
+            maxime nesciunt perspiciatis quaerat labore, quidem velit vel fugiat 
+            esse id? Veniam.
+         </p>
+         <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus voluptatem placeat blanditiis
+            at et aliquid architecto adipisci ipsa 
+            maxime nesciunt perspiciatis quaerat labore, quidem velit vel fugiat 
+            esse id? Veniam.
+         </p>
+         <blockquote>“I wish for a world that views disability, mental or physical, not as a hindrance but as unique attributes 
+            that can be seen as powerful assets if given the right opportunities. Accessability in 
+            technology is a basic human right. Let us advance digital equity together. ”
+         </blockquote>
+         <span>&#10077;</span>
+         <i> Anywhere I see suffering, that is where I want to be, doing what I can</i>
+         <span>&#10080;</span>
+         <br>

```

Comment:
Prefer to use margin styling to handle spacing between elements rather than stacking `br` tags.